src/gadgets/mpt_update.rs

```diff
@@ -1286,12 +1277,6 @@ fn configure_code_size<F: FieldExt>(
     bytes: &impl BytesLookup,
     poseidon: &impl PoseidonLookup,
 ) {
-    cb.assert(
```

Comment:
ExtensionNew(Old) are already disallowed for CodeSizeExists in configure_extension_new(old).